src/otx/data/transform_libs/torchvision.py

```diff
@@ -815,7 +816,7 @@ def __init__(
         self,
         prob: float | Iterable[float] | None = None,
         direction: str | Sequence[str | None] = "horizontal",
-        is_numpy_to_tvtensor: bool = False,
+        is_numpy_to_tvtensor: bool = True,
```

Comment:
Will this change affect speed or accuracy? For example, in `src/otx/recipe/detection/dfine_x.yaml`, `is_numpy_to_tvtensor` was previously set to `False` as default in `RandomFlip`, but it’s now True. Will this have any impact?